lib/grammers-client/src/parsers.rs

```diff
@@ -117,12 +117,21 @@ pub fn parse_markdown_message(message: &str) -> (String, Vec<tl::enums::MessageE
             update_entity_len!(Italic(offset) => entities);
         }
 
-        // [text link](https://example.com)
+        // [text link](https://example.com) or [user mention](tg://user?id=12345678)
         Event::Start(Tag::Link(_kind, url, _title)) => {
-            push_entity!(MessageEntityTextUrl(offset, url = url.to_string()) => entities);
+            if url.starts_with("tg://user?id") {
```

Comment:
Please refactor this into a `const MENTION_URL_PREFIX` (name can be better).

Reply:
ok.